apps/api/Dockerfile

```diff
@@ -1,12 +1,18 @@
-# Base image
-FROM node:20-alpine@sha256:b5b9467fe7b33aad47f1ec3f6e0646a658f85f05c18d4243024212a91f3b7554
+# Build stage
+FROM node:20-alpine@sha256:b5b9467fe7b33aad47f1ec3f6e0646a658f85f05c18d4243024212a91f3b7554 AS builder
+WORKDIR /app
+COPY . .
+RUN npm install -g pnpm
```

Comment:
You should run this command first locally:
https://pnpm.io/cli/approve-builds

Reply:
how about    RUN pnpm install  --ignore-scripts  ?

Reply:
> how about    RUN pnpm install  --ignore-scripts  ?

PNPM 10 runs with this parameter by default.